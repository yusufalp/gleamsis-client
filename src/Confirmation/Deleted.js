import React from 'react';
import { Link } from 'react-router-dom';

class Deleted extends React.Component {
  render() {
    return (
      <div>
        <h2>Deleted Successfully</h2>
        <Link to={'/dashboard'}>Go to dashboard</Link>
      </div>
    )
  }
}

export default Deleted;