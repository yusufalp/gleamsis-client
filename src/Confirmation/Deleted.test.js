import React from 'react';
import ReactDOM from 'react-dom';
import Deleted from './Deleted';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';

describe('Deleted component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(
      <MemoryRouter>
        <Deleted />
      </MemoryRouter>, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders the UI as expected', () => {
    const tree = renderer
      .create(
        <MemoryRouter>
          <Deleted />
        </MemoryRouter>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
})