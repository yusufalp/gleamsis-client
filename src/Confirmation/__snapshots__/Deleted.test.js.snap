// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Deleted component renders the UI as expected 1`] = `
<div>
  <header>
    <h2>
      Deleted Successfully
    </h2>
    <a
      href="/dashboard"
      onClick={[Function]}
    >
      Go to dashboard
    </a>
  </header>
</div>
`;
